apps/withdraw-balancer-v2/src/utils/getUserLpMints.ts

```diff
@@ -0,0 +1,91 @@
+import { SupportedChainId } from "@cowprotocol/cow-sdk";
+import { gql } from "urql";
+import { thegraphClientsMap } from "./thegraphClientsMap";
+
+export interface UserLpMintsReturn {
+  mints: {
+    sender: string;
+    to: string;
+    pair: {
```

Comment:
I am not familiar with this subgraph but make sure it handles the transfer case that I mentioned before.